### ISSUE TITLE
[Feature request]"Fog of war"

### ISSUE THREAD

**Issue by Lunat1q** [AHAAAAAAA/PokemonGo-Map#522]
I don't know  is it possible to do on google map, but here is it:
Add feature to see: 
1. What zone is currently in detection radius
2. Which "squares" was checked, and which wasn't
Imo it should looks like this:
http://i.imgur.com/0HTZTt3.png
where noColor - zone out of radius
blurred - not checked zone yet.

ofc this is just a picture, but idea is just showing those zones, no matter how :)

**Comment by KevakaBuel** [comment]
I would prefer a less drastical change, something like the range indicator you have inside the actual game. Feature itself would be pretty nice. #494 has a similar idea

**Comment by sykhro** [comment]
The blurring is a good design solution, we will make a few tests

**Comment by frdmn** [comment]
I totally agree with @KevakaBuel, this looks too prominent - Why not a simple translucent radius like this: 

![bildschirmfoto 2016-07-20 um 14 21 38](https://cloud.githubusercontent.com/assets/944459/16986127/4957303c-4e85-11e6-9b67-b57485079da9.png)

(via http://obeattie.github.io/gmaps-radius/?lat=51.500358&lng=-0.125506&z=10&u=mi&r=5)

**Comment by Lunat1q** [comment]
i didn't say it should be exactly as i draw. There should be 2 options: displaying detection zone, and displaying already scanned area INSIDE that zone.

**Comment by KevakaBuel** [comment]
I guess it cant be a circle because we scan in squares but i like the style of it =)

**Comment by DerScreever** [comment]
like @KevakaBuel  say, we scan in squares so a radius would confuse the user :D

**Comment by frdmn** [comment]
@DerScreever @KevakaBuel Oh, I was not aware of that. In this case we definitely should show a rectangle shape as well. 👍

**Comment by MikeDX** [comment]
We scan in squares but the radius of the scan is whatever niantic set, (distance from player) which I'm fairly sure is not a square box surrounding the player

**Comment by HinoReo** [comment]
@DerScreever I think that "we scan in squares" meant that the figure made of circles is square(ish), Though with implementation of hexagonal search pattern it will be more round, rendering a circle more appropriate, imo.
